cypress/integration/spec_devices.js

```diff
@@ -15,8 +15,8 @@ describe('Test adding different devices', () => {
         cy.get('#electrum_device_card').click()
         cy.get('#device_name').type("Electron's Electrum Device")
         // Open and close the explainer
-        cy.get('#toggle-explainer').click()
-        cy.get('#toggle-explainer').click()
+        cy.get("text-explainer").shadow().find('#toggle-explainer').click({force: true})
```

Comment:
I'd prefer to include **Cypress.config('includeShadowDom', true)** as below, then you don't need `.shadow()`
```
describe('Test adding different devices', () => {
    before(() => {
        cy.viewport(1200,660)
        cy.visit('/')
        Cypress.config('includeShadowDom', true)
    })
```

Reply:
Do you want to change it? @b30wulffz I have the feeling that above is more stable.

Reply:
I've simplified the test in https://github.com/cryptoadvance/specter-desktop/pull/1752/commits/2acb643fb6fd1e60e4711b3966976cbac773ebab
What do you think?

Reply:
Ahh I see, understood where I went wrong.